common/test/acceptance/pages/lms/teams.py

```diff
@@ -19,3 +28,27 @@ def is_browser_on_page(self):
     def get_body_text(self):
         """ Returns the current dummy text. This will be changed once there is more content on the page. """
         return self.q(css='.page-content-main').text[0]
+
+    def browse_topics(self):
+        """ View the Browse tab of the Teams page. """
+        self.q(css=BROWSE_BUTTON_CSS).click()
+
+    def get_topic_cards(self):
+        """Return a list of the topic cards present on the page."""
+        return self.q(css=TOPIC_CARD_CSS).results
+
+    def get_pagination_text(self):
+        """Return the text showing which topics the user is currently viewing."""
+        return self.q(css=PAGINATION_TEXT_CSS).text[0]
+
+    def get_current_page_text(self):
```

Comment:
Nit: this is confusingly named - it returns the numerical value of the current page as a string, no? Maybe you could cast to an int, and then rename to `get_current_page_number`.